src/model/StopPoint.ts

```diff
@@ -24,4 +24,17 @@ export const stopPointToPayload = (stopPoint: StopPoint) => {
   return rest;
 };
 
+export const flexibleStopPointToPayload = (stopPoint: StopPoint) => {
```

Comment:
I'm not sure what's going on here, can you elaborate?

Reply:
There are occasions when these keys get removed (in MixedFlexibleStopPointsEditor and MixedFlexibleStopPointEditor), and backend would throw an error if `flexibleStopPlaceRef: null` isn't explicitly supplied for non-flexible stop point. The other `rest['quayRef'] = null;` didn't seem to be needed (at least just didn't find a case when it would be) but added it just in case still, because before those field deletions were added in MixedFlexibleStopPoint(-s)Editor that's how the payload looked